README.md

```diff
@@ -37,7 +37,7 @@ echo "define(\`beep', \`boop')dnl\nbeep\n" | m4
 var M4 = require('m4');
 
 var input = new M4();
-input.pipe(M4()).pipe(process.stdout);
+input.pipe(new M4()).pipe(process.stdout);
```

Comment:
actually I believe it should just be:

     var m4 = new M4();
     m4.pipe(process.stdout);